skimage/__init__.pyi

```diff
@@ -39,3 +39,18 @@ from . import (
     transform,
     util,
 )
+
+# Legacy imports into the root namespace; not advertised in __all__
```

Comment:
But, presumably these *will* now be advertised in `__all__`?

Reply:
Done.